frontend/package.json

```diff
@@ -14,6 +15,9 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.13.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
```

Comment:
don't you want to _remove_ some font awesome deps instead of adding some — since you're using a CDN?

Reply:
yep I was changing that but I had my call with Geoffrey meanwhile

Reply:
Gonna update that right now